meson.build

```diff
@@ -0,0 +1,153 @@
+project('geowombat', ['c', 'cython'])
+
+py_mod = import('python')
+py3 = py_mod.find_installation('python3')
+
+# basic files
+
+py3.install_sources(
+    [
+    'src/geowombat/__init__.py',
+    'src/geowombat/config.py',
+    'src/geowombat/config.ini',
+    'src/geowombat/handler.py',
+    ],
+    subdir: 'geowombat',
+    pure: false
```

Comment:
Since meson 0.64, you can use `pure: false` in "find_installation" and it will be the defaulted value for each "install_sources" call, which lets you save a bit on DRY.